syntax = "proto3";

option java_multiple_files = true;
option java_package = "br.com.zup.grpc";
option java_outer_classname = "HelloWorldProto";
option objc_class_prefix = "HLW";

package br.com.zup.grpc;

service HelloService {
  rpc hello (HelloRequest) returns (HelloResponse) {}
}

message HelloRequest {
  string firstName = 1;
  string lastName = 2;
  string from = 3;
}

message HelloResponse {
  string message = 1;
}